core/templates/dev/head/components/social_buttons/social_buttons_directive.html

```diff
@@ -1,21 +1,36 @@
 <script type="text/ng-template" id="components/socialButtons">
+<style type="text/css">
+  
+i.oppia-share-btn-fa {
+    display: inline-block;
```

Comment:
Pushed the another position and commit again.Kindly, do review it. Hope this time its all set.